kuma/users/tests/test_templates.py

```diff
@@ -505,27 +541,24 @@ def test_not_banned_user_one_click_page_template_ban_button(self):
         eq_(ban_button.text(), "Ban User for Spam")
         eq_(len(ban_other_reasons), 0)
 
```

Comment:
Thanks for thinking this through and my apologies for the confusion.
I meant to create some revisions by `self.testuser2`, then to mark them as spam, and then to go to the one-click banning page for `self.testuser2` and to check the text of the ban button, which should say `"Ban User for Spam"`. I've updated the test. Let me know if you still think it needs work.